libpcan/CMakeLists.txt

```diff
@@ -1,35 +1,32 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(libpcan)
 
-find_package(catkin REQUIRED COMPONENTS rosbuild roslib mk rosbuild rospack)
+find_package(catkin REQUIRED COMPONENTS mk rospack)
```

Comment:
Yes, `mk` is a catkin package. I think we would not even need to find_package it, as it is not required for the build itself (i.e. no compiler flags), but used for downloading, extracting and building (via a MakeFile) the tarball with the sourcecode.